src/scene/shader-lib/chunks/lit/frag/TBNfast.js

```diff
@@ -1,5 +1,13 @@
 export default /* glsl */`
 void getTBN(vec3 tangent, vec3 binormal, vec3 normal) {
-    dTBN = mat3(tangent, binormal, normal);
+    #ifdef TWO_SIDED_LIGHTING
+        dTBN = mat3(
+            (gl_FrontFacing ? tangent  : -tangent ) * twoSidedLightingNegScaleFactor,
+            (gl_FrontFacing ? binormal : -binormal) * twoSidedLightingNegScaleFactor,
+            (gl_FrontFacing ? normal   : -normal  ) * twoSidedLightingNegScaleFactor
+        );
```

Comment:
Could simplify to something like
```suggestion
    dTBN = mat3(tangent, binormal, normal) * (-float(gl_FrontFacing) * twoSidedLightingNegScaleFactor);
```

Reply:
I'd vote for:

```
    dTBN = mat3(tangent, binormal, normal);

    #ifdef TWO_SIDED_LIGHTING
    dTBN *= ((gl_FrontFacing ? 1.0 : -1.0) * twoSidedLightingNegScaleFactor);
    #endif
```

Reply:
Fixed.